controllers/release/release_adapter_test.go

```diff
@@ -259,11 +259,7 @@ var _ = Describe("Release Adapter", Ordered, func() {
 		Expect(adapter.targetContext).To(Equal(adapter.context))
 	})
 
-	It("ensures the target context is set when a workspace is targeted by releaseplan", func() {
```

Comment:
@johnbieren 
I understood that it should fail if it is a non-KCP env but the Workspace is set as in @davidmogar 's comment:

>3. Based on the previous information we will validate if the workspace should be "" (non-KCP) or have an actual value (KCP) and fail if necessary (abort other operations).

By doing this the test can't work because we are not able to set the context for an inexistent Workspace.

Reply:
Yes, I agree it should fail in this scenario. Shouldn't this change introduce a test for that scenario?

Reply:
Yes. I have done it but noticed now that after the rebase it went to hell. Probably I removed it wrongly to resolve the conflicts.

Reply:
test added.

Reply:
I see the new test being added now, but I still see a test being removed. We already had a unit test that ensures the target context is properly modified when a workspace is defined. It isn't real KCP, but with the functions available, we could get it close enough for a working test. I do not know why we are removing this test. We still need to ensure that when a workspace is defined, the target context is modified appropriately. This is a very important part of the code to keep working